repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/keystonev2/timeout/diffpool/http-connection-pool.cfg.xml

```diff
@@ -1,27 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <http-connection-pools xmlns="http://docs.openrepose.org/repose/http-connection-pool/v1.0">
     <pool id="default"
-          default="false"
+          default="true"
```

Comment:
why were these swapped?

Reply:
Hah, I don't think it was swapped. I think GitHub is confused about what file was moved. The "original" file (on master) would be:
https://github.com/rackerlabs/repose/blob/master/repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/keystonev2/akkatimeout/diffpool/http-connection-pool.cfg.xml